test/neura/fusion/kernel.cpp

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#define NTAPS 1024
+
+int A[NTAPS][NTAPS];
+int s[NTAPS];
+int q[NTAPS];
+int p[NTAPS];
+int r[NTAPS];
```

Comment:
Meaningless kernel with meaningless variables? Just to verify the fusion functionality, right?